zboxcore/sdk/commitworker.go

```diff
@@ -117,18 +117,14 @@ func (commitreq *CommitRequest) processCommit() {
 			return err
 		}
 		defer resp.Body.Close()
-		if resp.StatusCode != http.StatusOK {
```

Comment:
Are all non-OK responses contain a body?
I think that was the reason there were this early copy of the check.